src/test/java/ParserTest.java

```diff
@@ -0,0 +1,17 @@
+import Grapie.Functions.Parser;
+import Grapie.Functions.Ui;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+
+    @Test
+    public void testMakeSenseOfUserCommand() {
+        assertEquals(Ui.LIST, new Parser().makeSenseOfUserCommand("list"));
```

Comment:
Do you have a `makeSenseOfUserCommand` method in your codebase? this seems like a dummy test.

Reply:
If you do have a `makeSenseOfUserCommand`, you may want to rename it to something better like `parseCommand`